AFQ/segmentation.py

```diff
@@ -42,7 +43,9 @@ def __init__(self,
                  b0_threshold=0,
                  prob_threshold=0,
                  rng=None,
-                 return_idx=False):
+                 return_idx=False,
+                 filter_by_endpoints=True,
+                 dist_to_aal=4):
```

Comment:
@jyeatman : in your experience, what is a reasonable distance to use for filtering? Using this distance still seems to leave quite a few streamlines that look spurious (at least for CST, at least in the example dataset).

Reply:
I would have suggested 4mm just looking at fiber endpoints and atlas voxels. The atlas is very rough and just serves to clear up the really bad streamlines

Reply:
Great - that helped - calibrating to the voxel size, so the distance is in mm, solved the particular issue I was seeing with that CST, so 4 mm does seem to be a good way to go (but 8 mm is too much 😄)